packages/transit-vehicle-overlay/src/index.js

```diff
@@ -80,18 +102,21 @@ class Vehicles extends MapLayer {
 
   /**
    * set zoom, used for changing icons ..
-   * note: limit (throttle) so we don't overwhelm React with tons of state changes zoomming in
+   * note: limit (throttle) so we don't overwhelm React with tons of state changes zooming in
    */
   setMapZoom = throttle(500, zoom => {
     this.setState({ mapZoom: zoom });
+    this.recenterMap();
   });
 
   /** pan the map to the tracked vehicle's coordinates */
   recenterMap = () => {
```

Comment:
Is there a way for the overlays to not control the map?

Reply:
Hey Binh. yeah this is a bit of a work-around. Will investigate how to simplify the zoom/panTo on the RT Vehicles. Will work to understand why map zooming and panTo (i.e., recenterMap()) need such special handling.  See: https://github.com/opentripplanner/otp-ui/issues/71

Reply:
BTW, thanks for the review.